raft/raft.go

```diff
@@ -117,12 +117,6 @@ type Config struct {
 	// ID is the identity of the local raft. ID cannot be 0.
 	ID uint64
 
-	// peers contains the IDs of all nodes (including self) in the raft cluster. It
-	// should only be set when starting a new raft cluster. Restarting raft from
-	// previous configuration will panic if peers is set. peer is private and only
-	// used for testing right now.
-	peers []uint64
-
 	// learners contains the IDs of all learner nodes (including self if the
```

Comment:
Please squash the commits such the "learners" are removed here as well.

Reply:
done